src/vs/base/parts/quickinput/browser/media/quickInput.css

```diff
@@ -126,6 +126,11 @@
 	padding: 5px 5px 2px 5px;
 }
 
+.quick-input-message > .codicon {
+	margin: 0 0.2em;
+	vertical-align: text-bottom;
```

Comment:
Why text-bottom:

text-bottom:
![image](https://user-images.githubusercontent.com/30295578/87878884-b98e1d80-ca04-11ea-892d-ea1073e0ee81.png)

middle:
![image](https://user-images.githubusercontent.com/30295578/87878889-c14dc200-ca04-11ea-942d-6587227c655d.png)

flex (works only for singleline):
![image](https://user-images.githubusercontent.com/30295578/87878898-cf034780-ca04-11ea-961d-402020518151.png)